### DISCUSSION TITLE
[Vote ended on 2023-03-09] Start implementing Semantic Markup in Ansible Documentation

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#205] [Votes done]
Discussion: #53, in particular https://github.com/ansible-community/community-topics/issues/53#issuecomment-1441965593
Proposal for semantic markup: https://hackmd.io/VjN60QSoRSSeRfvGmOH1lQ
PRs for ansible-doc/validate-modules and antsibull-docs: https://github.com/ansible/ansible/pull/74937 https://github.com/ansible-community/antsibull-docs/pull/4
Example PRs with how semantic markup looks: https://github.com/ansible-collections/community.dns/pull/23

This vote is about the following proposal:

> Merge the antsibull-docs PR to enable semantic markup in the docsite, and try to get ansible-core to accept the ansible-doc/validate-modules PR (or something equivalent) so that ansible-doc correctly renders semantic markup from ansible-core 2.15 on.

Please vote +1 if you are in favor of this, and -1 if you want this to not happen (at least not now).

**Comment by felixfontein** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by oraNod** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by acozine** [comment]
+1 this has been a long time coming

**Comment by tremble** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by jamescassell** [comment]
+1

**Comment by bluikko** [comment]
+1

**Comment by gundalow** [comment]
+1
